webpack.config.js

```diff
@@ -81,6 +78,10 @@ const config = {
 			// There are currently resolution errors on RSF's "mitt" dependency
 			// when imported as native ES module
 			'react-slot-fill': 'react-slot-fill/lib/rsf.js',
+			react: 'preact-compat',
+			'react-dom': 'preact-compat',
+			// Not necessary unless you consume a module using `createClass`
+			'create-react-class': 'preact-compat/lib/create-react-class',
```

Comment:
Is this needed for the components we're using?

Reply:
it maybe needed for the react libraries we're using but yeah we could try to remove it and see